### PR TITLE
Tiwariav task/3756/landing billing page

### PR DESCRIPTION
**Pull request by pcoder**
This is an extension of PR #480, where we create a `CustomUser` at the very end of the flow.

Some notes on this:

Several possible cases in the flow are:

- User is already logged in at the beginning of the flow: In this case, we create the VM using this (logged in user). On completion of the flow, the user is taken back to the dashboard.
- User is not logged in during the entire flow, but is already registered: We create the VM with the user account associated with the given email. At the end of the flow after the user is shown the 'thank you' modal, he/she is redirected to Datacenterlight index page.
- User is not logged in and not registered: We create the user account, only after the payment is successful and create VM with this new user. Previously, it was created using oneadmin user and later assigned to the corresponding user manually. After the user account is created, we send user activation email (as before) to the user alongwith the credentials (new in this PR). On completion of the flow, the user is authenticated, logged in and taken to the dashboard.

### CODE REVIEW

File: datacenterlight/locale/de/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -479,11 +511,23 @@ msgstr "Ungültige RAM-Grösse"
 msgid "Invalid storage size"
 msgstr "Ungültige Speicher-Grösse"
 
-msgid "is not a proper name"
-msgstr "ist kein gültiger Name"
+msgid "Another user exists with that email!"
+msgstr ""
```
Comment (verysanghee) [inline datacenterlight/locale/de/LC_MESSAGES/django.po]: "Ein Benutzer mit dieser E-Mail existiert bereits."

File: hosting/locale/de/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -286,10 +286,10 @@ msgstr ""
 "%(base_url)s%(my_virtual_machines_url)s\n"
 
 msgid "Toggle navigation"
```
Comment (verysanghee) [inline hosting/locale/de/LC_MESSAGES/django.po]: What is "Toggle navigation" for? Trying to understand the relation with "Konfiguration/Umschalten."

Comment (pcoder) [reply hosting/locale/de/LC_MESSAGES/django.po]: That seems to come from the navbar, probably @tiwariav can have an explanation for that

File: datacenterlight/locale/de/LC_MESSAGES/django.po (hunk 1 above)
Comment (pcoder) [reply datacenterlight/locale/de/LC_MESSAGES/django.po]: That text does not exist now.

**Comment by verysanghee** [comment]
works nicely.

**Comment by verysanghee** [comment]
screen shot 2017-09-28 at 01 09 26 alignment needs to be fixed :)

**Pull request merged**